go/enclave/storage/storage.go

```diff
@@ -266,10 +266,14 @@ func (s *storageImpl) HealthCheck() (bool, error) {
 	defer s.logDuration("HealthCheck", measure.NewStopwatch())
 	headBatch, err := s.FetchHeadBatch()
 	if err != nil {
-		s.logger.Info("HealthCheck failed for enclave storage", log.ErrKey, err)
```

Comment:
Why was this log removed?

Reply:
It's a duplicate log, it logs the error downstream